gobblin-api/src/main/java/gobblin/writer/WriterOutputFormat.java

```diff
@@ -21,6 +21,7 @@
   AVRO("avro"),
```

Comment:
Slightly orthogonal but I am not sure what the purpose of this enum is. If anything, it should be more of a registry where different writers can register they formats.